site/src/components/Navbar/NavbarView.tsx

```diff
@@ -269,6 +269,23 @@ const ProxyMenu: FC<{ proxyContextValue: ProxyContextValue }> = ({
         onClose={closeMenu}
         sx={{ "& .MuiMenu-paper": { py: 1 } }}
       >
+        <MenuItem
+          sx={[
+            { fontSize: 14 },
+            { "&:hover": { backgroundColor: "transparent" } },
+            { wordWrap: "break-word" },
+            { inlineSize: "200px" },
+            { whiteSpace: "normal" },
+            { textAlign: "center" },
+          ]}
+          onClick={(e) => {
+            // Stop the menu from closing
+            e.stopPropagation()
+          }}
+        >
+          To improve workspace connections, select the closest region.
```

Comment:
```suggestion
          Reduce workspace latency by selecting the region nearest you.
```
Maybe put in a help tooltip next to that "Only applies to web connections. Local connections will automatically select the nearest region based on latency."